components/WeatherCard.tsx

```diff
@@ -0,0 +1,116 @@
+import { FC } from 'react';
+import { useSiteData } from '~/lib/hooks';
+import useTime from '~/lib/hooks/useTime';
+
+import { CloudyIcon, PartlyCloudyIcon, SunnyIcon } from './icons';
+import { getTotalExpectedOutput } from './dashboard-modules/ExpectedTotalOutput';
+import { generationDataOverDateRange } from '~/lib/graphs';
+import { GenerationDataPoint } from '~/lib/types';
+import NumberDisplay from './dashboard-modules/NumberDisplay';
+
+type WeatherProps = {
+  siteUUID: string;
+};
+
+const GetExpectedOutputOverDay = (
+  duskTime: Date,
+  dawnTime: Date,
+  offset: number,
+  forecastValues: GenerationDataPoint[],
+  clearSkyEstimate: GenerationDataPoint[]
+) => {
+  const newDuskTime = new Date(duskTime.getTime() + offset * 3600000);
+  const newDawnTime = new Date(dawnTime.getTime() + offset * 3600000);
```

Comment:
can we make a constant above all this called `millisInHour`, set it equal to 3600000?

Reply:
This can also be imported from `date-fns` as a constant I think.